cross/json-c/patches/remove-variable-size.patch

```diff
@@ -0,0 +1,15 @@
+--- json_tokener.c.orig	2014-04-11 04:35:45.000000000 +0200
++++ json_tokener.c	2014-11-19 15:08:24.483987405 +0100
```

Comment:
Please add comment with patch source and reason to apply it.
Also start file name with number to enforce ordering

Reply:
ok, (took this untouched from cytec)